install.sh

```diff
@@ -0,0 +1,29 @@
+# Test for root
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root (sudo)"
+  exit
+fi
+
+# Copying the agent to its final destination
+cp -r jumper-logging-agent /opt
+cd /opt/jumper-logging-agent
+
+# Install pip and virtualenv
+apt-get install -y python-pip
+yes w | pip install virtualenv
+
+# Create a virtual environment and install the app
+virtualenv venv
+source venv/bin/activate
+python setup.py install
+deactivate
+
+# Setup the jumper agent service
+cp jumper-agent.template jumper-agent.service
+echo "ExecStart=$PWD/venv/bin/python2.7 $PWD/agent_main.py" >> jumper-agent.service
```

Comment:
Seems to work, but I changed it to /opt/jumper-logging-agent anyhow given the change directory change.